ports/linux/datetime-init.c

```diff
@@ -88,6 +88,8 @@ bool datetime_local(
     int32_t to;
 
     if (gettimeofday(&tv, NULL) == 0) {
+        /* Needed for syncing of Datetime between NMC & BACnet */
+        tv.tv_sec += *utc_offset_minutes * 60;
         to = Time_Offset;
```

Comment:
This value doesn't look right. Why are we setting `to` to Time_Offset? It's not used anywhere else. I think this needs another look.